### PR TITLE
Vibed

### PR DESCRIPTION
**Pull request by msmith-techempower**
Introduces `vibed` based on the initial commit from @nin-jin in #1597 

This simply fixed the minor bugs and removed the `install.sh` file.

### CODE REVIEW

File: toolset/setup/linux/prerequisites.sh [hunk 1]
```diff
@@ -45,7 +45,8 @@ sudo apt-get -qqy install -o Dpkg::Options::="--force-confdef" -o Dpkg::Options:
   libsasl2-dev                      `# Needed by mgo for go-mongodb test` \
   llvm-dev                          `# Required for correct Ruby installation` \
   libboost-dev                      `# Silicon relies on boost::lexical_cast.` \
-  postgresql-server-dev-9.3         `# Needed by cpoll.`
+  postgresql-server-dev-9.3         `# Needed by cpoll.`\
+  xdg-utils                         `# Needed by dlang.`
```
Comment (methane) [inline toolset/setup/linux/prerequisites.sh]: I think xdg-utils is not necessary.
@nin--jin said
> Note that the line sudo apt-get install -qq -y g++ gcc-multilib xdg-utils shouldn't be necessary.

File: frameworks/D/vibed/source/app.d [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	auto router = new URLRouter;
+	router.get("/plaintext", &plaintext);
+	router.get("/json", &json);
+	router.get("/db", &db);
+	router.get("/queries", &queries);
+	router.get("/generate-world", &generateWorld);
+	router.get("/generate-fortunes", &generateFortunes);
+	router.get("/", staticTemplate!"index.dt");
+
+	auto settings = new HTTPServerSettings;
```
Comment (Dith3r) [inline frameworks/D/vibed/source/app.d]: Did you test adding distribute option?
```
settings.options |= HTTPServerOption.distribute;
```

File: toolset/setup/linux/prerequisites.sh (hunk 1 above)
Comment (msmith-techempower) [inline toolset/setup/linux/prerequisites.sh]: Good point; I'll try removing it from my system and re-running the test.

File: frameworks/D/vibed/source/app.d (hunk 2 above)
Comment (msmith-techempower) [inline frameworks/D/vibed/source/app.d]: No, I am simply getting it to the point where it works in the suite. After this gets merged in (soon), others can open pull request to fine-tune it.

File: toolset/setup/linux/prerequisites.sh (hunk 1 above)
Comment (msmith-techempower) [inline toolset/setup/linux/prerequisites.sh]: Checked - it is required.

**Comment by msmith-techempower** [comment]
Well... travis failed. However, I have re-tested this on my local several times and it just works... not sure what the deal is... inclined to merge as-is.

**Pull request merged**